src/orchestratord/src/bin/orchestratord.rs

```diff
@@ -45,8 +50,159 @@ pub struct Args {
     #[clap(long, default_value = "[::]:3100")]
     metrics_listen_address: SocketAddr,
 
-    #[clap(flatten)]
-    materialize_controller_args: controller::materialize::MaterializeControllerArgs,
+    #[clap(long)]
+    cloud_provider: CloudProvider,
+    #[clap(long)]
+    region: String,
+    #[clap(long)]
+    create_balancers: bool,
+    #[clap(long)]
+    create_console: bool,
+    #[clap(long)]
+    helm_chart_version: Option<String>,
+    #[clap(long, default_value = "kubernetes")]
+    secrets_controller: String,
+    #[clap(long)]
+    collect_pod_metrics: bool,
+    #[clap(long)]
+    enable_prometheus_scrape_annotations: bool,
+    #[clap(long)]
+    disable_authentication: bool,
+
+    #[clap(long)]
+    segment_api_key: Option<String>,
+    #[clap(long)]
+    segment_client_side: bool,
+
+    #[clap(long)]
+    console_image_tag_default: String,
+    #[clap(long)]
+    console_image_tag_map: Vec<KeyValueArg<String, String>>,
+
+    #[clap(long)]
+    aws_account_id: Option<String>,
+    #[clap(long)]
+    environmentd_iam_role_arn: Option<String>,
```

Comment:
The refactor is fine, but it does remind me that we should we should be probalby have prefixed the flags with the cloud, iam_role_arn is pretty AWS specific, but being explicit that it's AWS would've been nice.